staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -426,7 +426,7 @@ func startCompactorOnce(c storagebackend.TransportConfig, interval time.Duration
 	}, nil
 }
 
-func newETCD3Storage(c storagebackend.ConfigForResource, newFunc, newListFunc func() runtime.Object, resourcePrefix string) (storage.Interface, DestroyFunc, error) {
+func newETCD3Storage(c storagebackend.ConfigForResource, newFunc, newListFunc func() runtime.Object, newTombFunc func(string) ([]byte, error), resourcePrefix string) (storage.Interface, DestroyFunc, error) {
```

Comment:
Not sure why we are creating tomb objects in etcd3 and not watch cache? Should watch cache be responsible for filling up prevKV?